src/main/java/compal/commons/Compal.java

```diff
@@ -0,0 +1,157 @@
+package compal.commons;
+
+import compal.model.tasks.Task;
+import compal.storage.StorageManager;
+import compal.ui.UiPart;
+import compal.logic.parser.ParserManager;
+import compal.storage.Storage;
+import compal.model.tasks.TaskList;
+import javafx.scene.paint.Color;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Comparator;
+import java.util.Date;
+
+
+/**
+ * Main class.
+ */
+public class Compal {
+
+    //***Class Properties/Variables***--------------------------------------------------------------------------------->
+    //objects supporting COMPal.Compal
+    public UiPart ui;
+    public Storage storage;
+    public TaskList tasklist;
+    public ParserManager parser;
+    //----------------------->
+
+    //***CONSTRUCTORS***------------------------------------------------------------------------------------------------
+    //------------------------------------------------------------------------------------------------------------------
+    //----------------------------------------------------------------------------------------------------------------->
+
+    /**
+     * Constructs Compal object.
+     * Initializes supporting objects.
+     * Starts off the parser CLI parsing loop.
+     */
+    public Compal() {
+        System.out.println("Compal:LOG: In Compal Constructor");
+        //Instantiate objects
+        tasklist = new TaskList(this);
+
+        storage = new StorageManager();
+
+        tasklist.arrlist = new ArrayList<>();
+        tasklist.arrlist = storage.loadCompal();
+
+        ui = new UiPart(this, tasklist.arrlist);
+
+        //start parsing commands
+        parser = new ParserManager(this, tasklist);
+    }
+    //----------------------->
+
+
+    //***MISC FUNCTIONS***----------------------------------------------------------------------------------------------
+    //------------------------------------------------------------------------------------------------------------------
+    //----------------------------------------------------------------------------------------------------------------->
+
+    //----------------------->
```

Comment:
As mentioned above, this definitely waste too many lines and makes code unnecessarily long and harder to read.